pkgs/applications/version-management/got/default.nix

```diff
@@ -17,12 +17,18 @@ stdenv.mkDerivation rec {
   ++ lib.optionals stdenv.isDarwin [ libossp_uuid ];
 
   preConfigure = lib.optionals stdenv.isDarwin ''
-    # The configure script assumes dependencies on Darwin are install via
+    # The configure script assumes dependencies on Darwin are installed via
     # Homebrew or MacPorts and hardcodes assumptions about the paths of
     # dependencies which fails the nixpkgs configurePhase.
     substituteInPlace configure --replace 'xdarwin' 'xhomebrew'
   '';
 
+  patches = [ ./darwin.patch ];
```

Comment:
Without this patch the buildPhase (linking) fails with missing symbol bsd_getopt

Reply:
Probably should use an `if` here, instead of lib.optional. It's now broken on Darwin :/